### PR TITLE
RHIDP-4805: RHBK v24 support for RHDH 1.4

### PR DESCRIPTION
**Pull request by hmanwani-rh**
**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):**
1.4

Related RN: 
**New Features**

Authenticating with Red Hat Build of Keycloak (RHBK) v24

With this update, you can use RHBK as an authentication provider. The Keycloak plugin will now support ingesting users and groups with RHBK. For more details, see link:https://docs.redhat.com/en/documentation/red_hat_build_of_keycloak/26.0/html-single/getting_started_guide/index#getting-started-zip-start-red-hat-build-of-keycloak[Authentication with RHBK].

**Deprecated Functionalities**

RHSSO 7.6 Deprecation

RHSSO 7.6 is deprecated as an authentication provider. You can continue to use RHSSO until the end of maintenance support. For details, see link:https://access.redhat.com/support/policy/updates/jboss_notes/#p_sso[RHSSO lifecycle dates]. As an alternative, migrate to RHBK v24.
Add the relevant labels to the Pull Request.
**Issue:**

### CODE REVIEW

**Review by themr0c — APPROVED** [review]

File: modules/authentication/proc-enabling-authentication-with-rhbk.adoc [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-* You have sufficient permissions in {rhsso} to create and manage a realm.
+* You have sufficient permissions in {rhbk} to create and manage a realm.
 
 .Procedure
-. To allow {product-short} to authenticate with {rhsso}, complete the steps in {rhsso}, to link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#realms-apps_[create a realm and a user] and link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#registering-app_[register the {product-short} application]:
+. To allow {product-short} to authenticate with {rhbk}, complete the steps in {rhbk}, to link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#realms-apps_[create a realm and a user] and link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#registering-app_[register the {product-short} application]:
```
Comment (kim-tsao) [inline modules/authentication/proc-enabling-authentication-with-rhbk.adoc:12]: Can you please replace the links with the ones from the RHBK docs?

https://docs.redhat.com/en/documentation/red_hat_build_of_keycloak/24.0/html/getting_started_guide/getting-started-zip-#getting-started-zip-create-a-realm

https://docs.redhat.com/en/documentation/red_hat_build_of_keycloak/24.0/html/getting_started_guide/getting-started-zip-#getting-started-zip-create-a-user

I'm not sure if this [topic](https://docs.redhat.com/en/documentation/red_hat_build_of_keycloak/24.0/html/getting_started_guide/getting-started-zip-#getting-started-zip-secure-the-first-application) is equivalent to the [RHSSO topic](https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#registering-app_).  @albarbaro can you confirm?

File: modules/authentication/proc-enabling-authentication-with-rhbk.adoc [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
 
 .. Use an existing realm, or link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#create-realm_[create a realm], with a distinctive **Name** such as __<my_realm>__.
 Save the value for the next step:
-* **{rhsso} realm base URL**, such as: __<your_rhsso_URL>__/auth/realms/__<your_realm>__.
+* **{rhbk} realm base URL**, such as: __<your_rhbk_URL>__/realms/__<your_realm>__.
 
-.. To register your {product-short} in {rhsso}, in the created realm, link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#registering-app_[create a Client ID], with:
+.. To register your {product-short} in {rhbk}, in the created realm, link:https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html-single/getting_started_guide/index#registering-app_[create a Client ID], with:
```
Comment (kim-tsao) [inline modules/authentication/proc-enabling-authentication-with-rhbk.adoc:18]: similar to the comment above, please replace the link

**Review by kim-tsao — COMMENTED** [review]
Looks good overall, just need to make sure the links are updated

**Review by albarbaro — APPROVED** [review]
Apart from the links Kim already mentioned, looks good to me.

File: modules/authentication/proc-enabling-authentication-with-rhbk.adoc (hunk 1 above)
Comment (hmanwani-rh) [reply modules/authentication/proc-enabling-authentication-with-rhbk.adoc]: Done

File: modules/authentication/proc-enabling-authentication-with-rhbk.adoc (hunk 2 above)
Comment (hmanwani-rh) [reply modules/authentication/proc-enabling-authentication-with-rhbk.adoc]: Done

**Pull request merged**